trax/data/tf_inputs.py

```diff
@@ -1264,6 +1265,61 @@ def split_stream(generator=None):
   return split_stream
 
 
+def LoadJSONRows(dir_path=gin.REQUIRED, fname=gin.REQUIRED):
+  path = os.path.join(dir_path, fname)
+  with open(path, 'r') as f:
+    # loads everything into memory
+    ds = [json.loads(line) for line in f.readlines()]
+
+  def iterator(generator=None):
+    while True:
+      for el in ds:
+        yield el
+  return iterator
+
+
+def AQuAExtendedQuestion():
```

Comment:
Can you add docstrings as in https://google.github.io/styleguide/pyguide.html or as visible in other places in TRAX? Also, we will need a short info how to work with the new loss and the new dataset.